ros2pkg/ros2pkg/api/create.py

```diff
@@ -89,180 +203,52 @@ def create_package_environment(package, destination_directory):
-        package_directory,
-        package.name + 'ConfigVersion.cmake.in',
-        version_config)
-
-
-def populate_ament_cmake(package, package_directory, cpp_node_name, cpp_library_name):
-    cmakelists_config = {
-        'project_name': package.name,
-        'dependencies': [str(dep) for dep in package.build_depends],
-        'cpp_node_name': cpp_node_name,
-        'cpp_library_name': cpp_library_name,
-    }
-    _create_template_file(
-        'ament_cmake/CMakeLists.txt.em',
-        package_directory,
-        'CMakeLists.txt',
-        cmakelists_config)
-
-
-def populate_cpp_node(package, source_directory, cpp_node_name):
-    cpp_node_config = {
-        'package_name': package.name,
-    }
-    _create_template_file(
-        'cpp/main.cpp.em',
-        source_directory,
-        cpp_node_name + '.cpp',
-        cpp_node_config)
-
-
-def populate_cpp_library(package, source_directory, include_directory, cpp_library_name):
-    class_name = cpp_library_name.replace('_', ' ').title()
-    class_name = ''.join(x for x in class_name if not x.isspace())
-    cpp_header_config = {
-        'package_name': package.name,
-        'library_name': cpp_library_name,
-        'class_name': class_name,
-    }
-    _create_template_file(
-        'cpp/header.hpp.em',
-        include_directory,
-        cpp_library_name + '.hpp',
-        cpp_header_config)
-
-    cpp_library_config = {
-        'package_name': package.name,
-        'library_name': cpp_library_name,
-        'class_name': class_name
-    }
-    _create_template_file(
-        'cpp/library.cpp.em',
-        source_directory,
-        cpp_library_name + '.cpp',
-        cpp_library_config)
-
-    visibility_config = {
-        'package_name': package.name.upper(),
-    }
-    _create_template_file(
-        'cpp/visibility_control.h.em',
-        include_directory,
-        'visibility_control.h',
-        visibility_config)
+    return package_directory
+
+
+def display_create_package_environment_message(args, package):
+    """
+    Print stdout message describing the package details to be created.
+
+    :param args: Commandline args wrapped by package_create_arguments object.
+    :param package: Package details.
+    """
+    print('create package')
+    print('  package name:', package.name)
+    print('  destination directory:', os.path.abspath(args.destination_directory))
+    print('  package format:', package.package_format)
+    print('  version:', package.version)
+    print('  description:', package.description)
+    print('  maintainer:', [str(maintainer) for maintainer in package.maintainers])
+    print('  licenses:', package.licenses)
+    print('  build type:', package.get_build_type())
+    print('  dependencies:', [str(dependency) for dependency in package.build_depends])
+    if args.node_name:
+        print('  node_name:', args.node_name)
+    if args.library_name:
+        print('  library_name:', args.library_name)
```

Comment:
We recommend the new `f-string` for print statement now. This is a good opportunity updating it. Example see https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498